src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -123,6 +191,14 @@ def __init__(self, target, options, credentials, compression):
         if compression:
             raise NotImplementedError("TODO: compression not implemented yet")
 
+        if interceptors is None:
+            self._unary_unary_interceptors = None
+        else:
+            self._unary_unary_interceptors = list(
+                filter(
+                    lambda interceptor: isinstance(interceptor, UnaryUnaryClientInterceptor),
+                    interceptors)) or None
```

Comment:
Instead of filter out silently, do you think we should raise an exception? Make least surprise?

Reply:
Agree on not filtering out silently, we can start by raising a `ValueError` if there is any filter that is not an instance of the types supported.